tasks/minsize/minsize.snap

```diff
@@ -11,7 +11,7 @@ Original   -> Minified   -> Gzip       Brotli
 
 555.77 kB  -> 274.97 kB  -> 91.38 kB   77.41 kB   d3.js
 
-977.19 kB  -> 456.46 kB  -> 123.82 kB  107.39 kB  bundle.min.js
+977.19 kB  -> 456.45 kB  -> 123.82 kB  107.40 kB  bundle.min.js
```

Comment:
lol there is a case hit? I wonder what the code is lol

Reply:
I only found the following code 🤣:

```js
typeof function () { }
```